frontend/public/components/default-resource.tsx

```diff
@@ -132,6 +136,36 @@ export const DefaultList: React.FC<TableProps & { kinds: string[] }> = (props) =
     );
   };
 
+  const TableRowForACRQ: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj, customData }) => {
```

Comment:
This doesn't belong in default-resource.tsx. Instead of changing default-resource, define a `AppliedResourceClusterQuota` list page and register it in resource-pages.ts.